backend/.env.example

```
@@ -171,3 +171,15 @@ WIKI_CONTENT_WRITE_BASE_URL=http://backend:8000
 # Format: {"dingtalk":{"appKey":"YOUR_APP_KEY","appSecret":"YOUR_APP_secret","operatorId":"YOUR_OPERATOR_ID","userMapping":{"baseId":"YOUR_BASE_ID","sheetId":"YOUR_SHEET_ID"}}}
 # See backend/app/services/tables/DATA_TABLE_CONFIG_EXAMPLE.md for detailed configuration guide
 DATA_TABLE_CONFIG={"dingtalk":{"appKey":"YOUR_APP_KEY","appSecret":"YOUR_APP_secret","operatorId":"YOUR_OPERATOR_ID","userMapping":{"baseId":"YOUR_BASE_ID","sheetId":"YOUR_SHEET_ID"}}}
+
+# Long-term memory configuration (mem0)
+# Enable/disable long-term memory feature (default: False)
+MEMORY_ENABLED=False
+# mem0 service base URL (default: http://localhost:8080)
+MEMORY_BASE_URL=http://localhost:8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MEMORY_BASE_URL key should go before the MEMORY_ENABLED key

(UnorderedKey)